recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/domd-h3ulcb.cfg

```diff
@@ -16,7 +16,7 @@ device_tree = "/xt/domd/domd.dtb"
 # Uncomment this when using nfs as a boot device
 #extra = "root=/dev/nfs nfsroot=192.168.1.100:/srv/domd,vers=3 ip=dhcp rw rootwait console=hvc0 cma=256M@1-2G pvrsrvkm.DriverMode=0"
 # Uncomment this when using block device as a boot device
-extra = "root=/dev/STORAGE_PART5 userdata=/dev/STORAGE_PART6 ro rootwait console=hvc0 cma=256M@1-2G pvrsrvkm.DriverMode=0"
+extra = "root=/dev/STORAGE_PART5 ro userdata=/dev/STORAGE_PART6 rootwait console=hvc0 cma=256M@1-2G pvrsrvkm.DriverMode=0"
```

Comment:
Looks like you just move ro to another place. What is the difference?

Reply:
No difference. Just thought placing ro near rootfs will make command args more obvious that ro is related to rootfs.